### ISSUE TITLE
Move focused view command is not accessible from Command Pallette

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#93974]
Testing #93786

`Move focused view` command is not accessible from Command Pallette. 

I see the issue with context keys.

How about having a command `Move view` which gives list of views to select and then you can move that?

**Comment by sbatten** [comment]
I personally don't see a huge benefit. This command should be bound if it is to be used regularly in which case the command palette is not necessary.

**Comment by sandy081** [comment]
This will be aligned with 

<img width="621" alt="image" src="https://user-images.githubusercontent.com/10746682/78047742-02986400-7379-11ea-9f2c-8f8a66a087b0.png">
